test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -1027,11 +1027,7 @@ class TestDapp {
   /**
    * Scrolls to the create token button and clicks it.
    */
-  async findAndClickCreateToken() {
-    const createTokenElement = await this.driver.findElement(
-      this.createTokenButton,
-    );
-    await this.driver.scrollToElement(createTokenElement);
```

Comment:
the find+scrolling action is unnecessary - our clickElement method handles everything that's needed for that element to be clicked